ports/esp32/network_ppp.c

```diff
@@ -266,6 +266,40 @@ static mp_obj_t ppp_ifconfig(size_t n_args, const mp_obj_t *args) {
 }
 MP_DEFINE_CONST_FUN_OBJ_VAR_BETWEEN(ppp_ifconfig_obj, 1, 2, ppp_ifconfig);
 
+static mp_obj_t ppp_ipconfig(size_t n_args, const mp_obj_t *args, mp_map_t *kwargs) {
+
```

Comment:
The PPP must be active, otherwise all requests fail. So better start this function with e.g.:
```
    ppp_if_obj_t *self = MP_OBJ_TO_PTR(args[0]);

    if (!self->active) {
        mp_raise_ValueError(MP_ERROR_TEXT("PPP not active"));
    }
```

Reply:
`ppp_ifconfig` also doesn't have such a guard, right?

Reply:
It does not, but it does also not crash with a core dump if not active. The guard here is to prevent a core dump crash. I assume it crashes here:
`struct netif *netif = ppp_netif(self->pcb);`

Reply:
aah right, the other path has an `self->pcb != NULL`. Then let's check for the same condition.

Reply:
Fine, then let's go for that test,